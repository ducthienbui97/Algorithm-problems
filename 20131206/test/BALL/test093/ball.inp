9 748245 412415 278537 639364
980098 24691 484272 435694 535874 824047 79284 621535 937688 380481 844197 67531 657657 358360 307457 165646 87105 217259 780010 378684 533783 381119 358644 377689 105468 937284 26655 707055 305354 566005 150789 66838 789122 634681 166797 281061 134935 764792 684457 205542 562985 414395 945866 471950 680491 684053 391179 333736 739259 596391 750367 158216 480816 803275 398815 762191 260195 481352 836181 256456 471322 966993 43010 7765 508469 652890 305564 889833 405116 134532 811096 939019 378533 938863 42669 379447 839910 270714 137983 217156 528882 
329299 271486 702109 625055 603594 478967 94682 779099 422829 555044 339840 604856 602520 73992 82319 629109 916452 878489 948883 677716 912109 820619 517933 728191 363115 925651 595507 987625 175979 268136 409388 242819 739012 711940 657246 348153 163516 438051 173121 544652 775978 948682 430081 418771 274034 684010 563690 947990 798883 290318 701768 565474 138073 802590 380029 740669 741396 420320 802350 822405 41151 426279 137418 415543 724510 526669 672343 716223 874562 641823 597248 818619 580377 752255 666029 270571 700419 940434 876338 799338 500711 
734640 859758 988346 518210 443230 514451 898465 374433 420555 283128 954802 570595 905655 846068 877566 568 710689 369222 611974 396340 392352 868056 169970 851683 892895 84813 991028 127633 554793 351387 743743 703599 341471 520141 617810 646744 278209 168632 85529 344528 140898 351581 708119 347864 228252 724288 778032 342506 346483 432734 852488 676712 9710 22146 963751 287407 606437 525428 766835 251321 660117 696461 266421 424333 302741 557372 303095 750151 526345 587398 200397 987047 381623 659220 756661 288538 400381 876363 587658 342851 314575 
259012 768841 924300 775328 73156 566145 560087 189887 348395 868163 715865 579250 207834 838283 797479 106071 533130 649461 376825 574969 573622 329355 890985 760084 814556 307279 364449 620026 652889 905616 643698 415909 953232 221560 157019 254510 380885 430817 684791 971492 398050 614196 333148 757495 763982 83384 573164 813139 824559 382874 881981 409037 50716 501460 52724 10828 729275 450773 820176 684285 865670 958087 351445 890774 934437 166124 167812 391926 504096 120595 448386 570163 151989 616445 487401 22302 520093 502419 999928 666788 263840 895952 514714 754697 747729 446826 483289 31931 788509 380869 634679 687897 516269 576008 713418 37229 69240 690633 191186 390739 769670 932550 423115 500869 72228 234870 912258 685758 949229 6793 180406 660693 416476 99419 545583 612499 705132 501385 741485 586159 398994 136257 465030 587027 643634 613201 283430 195243 620458 772710 711621 434813 291403 919182 469933 998051 663019 687304 956603 534947 116251 331849 28861 63573 145264 916860 467482 912858 375434 289286 23668 639951 336220 953713 420847 607586 203916 924113 21755 593633 758981 436746 681084 489370 841262 460966 397384 788613 908986 899583 706264 916497 19936 399147 786060 467009 163550 272320 704113 920639 344230 19446 423136 967778 325713 358606 773574 279332 587251 697164 469195 796195 875767 368709 856851 153166 605734 312191 303762 129103 677713 41302 810939 665327 856672 999438 779156 481588 483251 663909 61648 856091 330523 198556 239278 217341 592785 896095 209935 560721 897884 507456 814561 860944 721001 373635 813682 222475 8918 691150 276490 217719 854582 951398 591606 287274 308072 319374 707733 377017 712988 312176 928288 998893 409085 437058 867119 690375 322590 953223 237760 548661 70477 670103 423557 73876 642701 447310 197149 209932 679935 512010 783852 493806 31760 975792 968755 309067 712574 543803 575546 84807 39878 293463 422579 6565 132954 741923 585871 920242 888313 386798 156051 556448 12464 96470 66747 286433 431364 961037 957422 257754 974055 296867 586472 225877 310497 745491 410222 510441 772594 316772 325923 265944 301106 263600 987714 241724 769965 421026 512630 771204 692991 414836 853810 221558 370134 56784 73856 213435 903320 397345 323979 385120 185571 3323 310316 994590 186549 754314 899903 475094 471680 427628 535023 233964 201161 150365 723064 484806 742088 111972 5901 937310 464035 584674 786633 596293 209665 314820 330223 390437 558359 699901 167530 426794 363314 893827 852324 146304 878932 34134 657097 445940 497759 490697 183346 896432 513410 94304 453589 961938 957477 268095 676591 288914 968577 816155 736604 717731 181904 289358 788208 550996 86036 146938 769277 302416 975087 149840 332025 905519 293487 962378 310307 385136 943936 721307 641445 51473 487031 57682 816239 956015 359100 511833 558935 112678 699173 714869 679192 290701 348260 901581 35837 622802 219642 966087 586619 812290 745444 167240 550639 672520 288611 116682 530441 400755 723790 566769 544207 134959 698174 7598 239659 922299 987168 88952 546970 83605 339343 920805 262793 909295 722230 547076 529938 293324 239960 261971 392480 470875 944944 643781 648128 228693 973281 227751 525149 819458 276170 898481 604800 10533 294214 348374 57824 533063 370036 759251 87776 139019 15915 410777 305 717460 663524 533552 126109 953634 452312 677027 330412 127811 404153 488002 989108 597287 6731 458648 732527 868820 277100 936308 30142 157459 908814 513906 542815 155503 22819 35003 287437 345847 431671 503794 86529 651780 458507 534813 25430 399161 984433 546556 41035 115218 852897 328627 92333 17912 707393 100073 121685 291502 928105 895106 907454 831185 177010 838922 671036 822966 549172 175459 418951 805156 730781 916538 578722 574439 145937 219192 434 80333 853523 66310 930858 851570 711613 312269 944453 219647 799623 941402 523751 370518 340203 771630 566580 840346 783640 628953 730787 353746 760624 847138 334993 306825 193310 473864 370508 263654 116758 791591 279250 169045 613049 552101 871720 57279 809432 24395 175307 426625 223893 790040 936126 670535 415449 598672 826868 821775 778233 451724 471752 346831 865022 282596 798160 25650 845595 823678 159978 294305 989952 853179 889215 780129 551948 727957 864875 857972 289425 448209 83617 333774 994694 133010 957201 474056 941276 472750 884877 741813 455864 891716 784270 217450 316128 862211 115585 69830 904295 619091 539538 832881 140345 917554 817830 926388 345016 275398 874477 485682 221904 835894 482211 95092 220352 50642 183528 16463 984299 459082 367039 628137 984608 315063 361901 827201 79247 260175 578514 54366 549180 78535 282032 587064 229105 716145 355535 934874 808314 538183 706163 962814 605718 66921 612033 210672 107560 734139 882164 574784 658022 249292 618922 867548 523648 686724 170315 941104 884469 750316 271899 715123 823626 561119 998467 960569 341891 100381 661339 167662 618903 645425 301149 539878 443613 669448 722074 809268 399721 35014 63530 812173 52528 366038 585728 897907 562926 500550 774429 928193 289144 515613 433544 485885 984434 