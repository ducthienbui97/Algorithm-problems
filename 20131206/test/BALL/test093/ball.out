15060045691505
